build.sbt

```diff
@@ -915,3 +916,48 @@ lazy val runner = project
   .dependsOn(pkg)
   .dependsOn(`language-server`)
   .dependsOn(`polyglot-api`)
+
+lazy val launcher = project
+  .in(file("lib/scala/launcher"))
+  .configs(Test)
+  .settings(
+    libraryDependencies ++= Seq(
+        "org.scalatest" %% "scalatest" % scalatestVersion % Test,
+        "com.monovore"  %% "decline"   % declineVersion,
+        "org.typelevel" %% "cats-core" % catsVersion
+      )
+  )
+  .settings(
+    buildNativeImage := Def
+        .task {
+          val javaHome = System.getProperty("java.home")
+          val nativeImagePath =
+            if (sys.props("os.name").contains("Windows"))
+              s"$javaHome\\bin\\native-image.cmd"
+            else s"$javaHome/bin/native-image"
+          val classPath =
+            (Runtime / fullClasspath).value.files.mkString(File.pathSeparator)
+          val additionalParameters =
+            if (sys.props("os.name").contains("Linux"))
+              "--static"
+            else ""
+          val resourcesGlobOpt = "-H:IncludeResources=.*Main.enso$"
+          val cmd =
+            s"$nativeImagePath $additionalParameters $resourcesGlobOpt " +
+            s"--no-fallback --initialize-at-build-time" +
+            s" -cp $classPath ${(Compile / mainClass).value.get} enso"
+          if (cmd.! != 0) {
+            throw new RuntimeException("Native Image build failed")
+          }
+        }
+        .dependsOn(Compile / compile)
+        .value
+  )
+  .settings(
+    Compile / sourceGenerators += Def.task {
```

Comment:
This looks like copy-paste from `runner`.

Reply:
It is indeed.
The one in `runner` seems to not work anyway, so I'm not sure what criteria I should use when extracting it, but I think I can make this a separate module.